### ISSUE TITLE
AYA creates processes but does not open the main window

### ISSUE THREAD

**Issue by Null-Kelvin** [liriliri/aya#5]
Windows 10 22H2, AYA 0.1.0
After installation, AYA starts but no window appears. The AYA and adb processes are visible in Task Manager, but nothing else happens. 
![ProcessHacker_6O0viT4r4r](https://github.com/user-attachments/assets/64282e63-f1ab-473a-bb33-61bc469d2995)

**Comment by surunzi** [comment]
Run AYA.exe from terminal and see if there is any errors.

**Comment by Null-Kelvin** [comment]
Nope, nothing. But I manage to launch it with `AYA.exe --remote-debugging-port=8315 --remote-allow-origins=*`
When I added it address in `chrome://inspect/#devices` and inspect `%LOCALAPPDATA%/Programs/AYA/resources/app.asar/renderer/index.html` - main window appear and everything works. When I close chrome DevTools or disconnect inspect process - AYA process close itself without error.

**Comment by Null-Kelvin** [comment]
Funny thing. When I run procmon64 (Sysinternals) and add AYA.exe as a process of interest in the filters, AYA opens its main window.  If procmon is paused, AYA does not open the window again. @kojjii, can you confirm?
Version 0.1.1 have the same problem.

**Comment by KristoferYG** [comment]
same issue

**Comment by JoAl115** [comment]
I have the same issue on W11 24H2, but I didn't test with procmon, I will try tomorrow if it is to be any help. V0.1.1. tested

**Comment by emorevival** [comment]
windows 10 22h2, @Null-Kelvin's solution with procmon64 does fix the issue for me too. didn't need to do anything, just opened procmon64 then aya and the gui appeared successfully

**Comment by Null-Kelvin** [comment]
Version 0.2.0 still affected

**Comment by toolzgithub** [comment]
I also encountered this problem

**Comment by JoAl115** [comment]
My issue was gone after opening procmon once, still working after multiple reboots and also with v 0.2.0

**Comment by KEVIN970712** [comment]
same in 0.3.0

**Comment by surunzi** [comment]
I cannot reproduce the issue on all three of my Windows 10 machines. Could it be caused by antivirus software? Is there anyone who can verify this?

**Comment by Null-Kelvin** [comment]
I have no antivirus, even Windows Defender has been disabled. Interestingly, it is when trying to debug via ProcMon that everything works. And it's not just me.

**Comment by toolzgithub** [comment]
Use PowerShell to start AYA and print as follows
But the AYA window is not open
![Image](https://github.com/user-attachments/assets/ab13aa7b-7d7d-42fc-a270-bfd5a2c52eac)

**Comment by surunzi** [comment]
After reviewing the code, I suspect that Electron may not trigger the 'ready-to-show' event in certain situations. Can someone test this version to see if it fixes the issue?

https://release.liriliri.io/AYA-0.4.1-win-x64.exe

**Comment by toolzgithub** [comment]
> 在查看了代码后，我怀疑 Electron 在某些情况下可能不会触发 'ready-to-show' 事件。有人可以测试这个版本看看它是否解决了这个问题吗？
> 
> https://release.liriliri.io/AYA-0.4.1-win-x64.exe

**Comment by toolzgithub** [comment]
> After reviewing the code, I suspect that Electron may not trigger the 'ready-to-show' event in certain situations. Can someone test this version to see if it fixes the issue?
> 
> https://release.liriliri.io/AYA-0.4.1-win-x64.exe

It's perfect. After installing 0.4.1, the AYA window can open normally.

**Comment by KEVIN970712** [comment]
> After reviewing the code, I suspect that Electron may not trigger the 'ready-to-show' event in certain situations. Can someone test this version to see if it fixes the issue?
> 
> https://release.liriliri.io/AYA-0.4.1-win-x64.exe
yes! finally fix this problem,thank you

**Comment by surunzi** [comment]
Version 0.4.1 is released.
